lms/templates/ccx/enrollment.html

```diff
@@ -26,7 +27,7 @@
   </div>
 
   <div class="enroll-option">
-    <input type="checkbox" name="email-students" value="Notify-students-by-email" checked="yes">
+    <input type="checkbox" name="email-students" id="email-students" value="Notify-students-by-email" checked="yes">
```

Comment:
What does adding id have to do with a11y? (Honestly curious).  Also,doesn't this potentially change django form handling?